plugins/autopeering/autopeering.go

```diff
@@ -26,7 +26,7 @@ import (
 // autopeering constants
 const (
 	ProtocolVersion = 0      // update on protocol changes
-	NetworkVersion  = "v0.2" // update on network changes
```

Comment:
:thinking:  Initially it was the idea to have this consistent with the goshimmer version, but now we want to have a new network for each DB reset.

Should we maybe add a suffix to `v0.2`? Otherwise we could also remove all this `v0.` and use an integer

Reply:
I'll just change them both to '0.2.0'

Reply:
Actually no, I'll just make it a number